iset.mm

```diff
@@ -147339,6 +147339,448 @@ since the target of the homomorphism (operator ` O ` in our model) need
+    iscmnd.p $e |- ( ph -> .+ = ( +g ` G ) ) $.
+    iscmnd.g $e |- ( ph -> G e. Mnd ) $.
+    iscmnd.c $e |- ( ( ph /\ x e. B /\ y e. B ) -> ( x .+ y ) = ( y .+ x ) ) $.
+    $( Properties that determine a commutative monoid.  (Contributed by Mario
+       Carneiro, 7-Jan-2015.) $)
+    iscmnd $p |- ( ph -> G e. CMnd ) $=
+      ( cmnd wcel cv cfv co wceq wral oveqd raleqbidv eqid cplusg cbs wa 3expib
+      ccmn ralrimivv eqeq12d anbi2d mpbi2and iscmn sylibr ) AFKLZBMZCMZFUANZOZU
+      NUMUOOZPZCFUBNZQZBUSQZUCZFUELAULUMUNEOZUNUMEOZPZCDQZBDQZVBIAVEBCDDAUMDLUN
+      DLVEJUDUFAVGVAULAVFUTBDUSGAVEURCDUSGAVCUPVDUQAEUOUMUNHRAEUOUNUMHRUGSSUHUI
+      BCUSUOFUSTUOTUJUK $.
+  $}
+
+  ${
+    $d x y B $.  $d x y G $.  $d x y ph $.
+    isabld.b $e |- ( ph -> B = ( Base ` G ) ) $.
+    isabld.p $e |- ( ph -> .+ = ( +g ` G ) ) $.
+    isabld.g $e |- ( ph -> G e. Grp ) $.
+    isabld.c $e |- ( ( ph /\ x e. B /\ y e. B ) -> ( x .+ y ) = ( y .+ x ) ) $.
+    $( Properties that determine an Abelian group.  (Contributed by NM,
+       6-Aug-2013.) $)
+    isabld $p |- ( ph -> G e. Abel ) $=
+      ( cgrp wcel ccmn cabl grpmndd iscmnd isabl sylanbrc ) AFKLFMLFNLIABCDEFGH
+      AFIOJPFQR $.
+  $}
+
+  ${
+    $d x y B $.  $d x y G $.
+    isabli.g $e |- G e. Grp $.
+    isabli.b $e |- B = ( Base ` G ) $.
+    isabli.p $e |- .+ = ( +g ` G ) $.
+    isabli.c $e |- ( ( x e. B /\ y e. B ) -> ( x .+ y ) = ( y .+ x ) ) $.
+    $( Properties that determine an Abelian group.  (Contributed by NM,
+       4-Sep-2011.) $)
+    isabli $p |- G e. Abel $=
+      ( cabl wcel cgrp cv co wceq wral rgen2 isabl2 mpbir2an ) EJKELKAMZBMZDNUA
+      TDNOZBCPACPFUBABCCIQABCDEGHRS $.
+  $}
+
+  ${
+    $d x y B $.  $d x y G $.  $d x y .+ $.  $d x y W $.  $d x y X $.
+    $d x y Y $.  $d x y Z $.
+    $( A commutative monoid is a monoid.  (Contributed by Mario Carneiro,
+       6-Jan-2015.) $)
+    cmnmnd $p |- ( G e. CMnd -> G e. Mnd ) $=
+      ( vx vy ccmn wcel cmnd cv cplusg cfv co wceq cbs wral eqid iscmn simplbi
+      ) ADEAFEBGZCGZAHIZJRQSJKCALIZMBTMBCTSATNSNOP $.
+
+    ablcom.b $e |- B = ( Base ` G ) $.
+    ablcom.p $e |- .+ = ( +g ` G ) $.
+    $( A commutative monoid is commutative.  (Contributed by Mario Carneiro,
+       6-Jan-2015.) $)
+    cmncom $p |- ( ( G e. CMnd /\ X e. B /\ Y e. B ) ->
+                 ( X .+ Y ) = ( Y .+ X ) ) $=
+      ( vx vy ccmn wcel co wceq cv wral wa cmnd iscmn simprbi rsp2 imp caovcomg
+      sylan 3impb ) CJKZDAKEAKDEBLEDBLMUEHIDEABUEHNZINZBLUGUFBLMZIAOHAOZUFAKUGA
+      KPZUHUECQKUIHIABCFGRSUIUJUHUHHIAATUAUCUBUD $.
+
+    $( An Abelian group operation is commutative.  (Contributed by NM,
+       26-Aug-2011.) $)
+    ablcom $p |- ( ( G e. Abel /\ X e. B /\ Y e. B ) ->
+                 ( X .+ Y ) = ( Y .+ X ) ) $=
+      ( cabl wcel ccmn co wceq ablcmn cmncom syl3an1 ) CHICJIDAIEAIDEBKEDBKLCMA
+      BCDEFGNO $.
+
+    $( Commutative/associative law for Abelian groups.  (Contributed by NM,
```

Comment:
Abelian groups -> commutative monoids (also in the following)

Reply:
Fixed in set.mm and iset.mm for cmn32 , cmn4 , and cmn12